.github/docker/Dockerfile.swift

```diff
@@ -1,3 +1,3 @@
-FROM swift:5.10.1-jammy
+FROM swift:5.10.1-bookworm
```

Comment:
Switched to build on the same version that we run on. This doesn't appear to have an impact though since we link the plugins statically.